recipe/meta.yaml

```diff
@@ -113,9 +118,11 @@ outputs:
       summary: The postgres runtime libraries and utilities (not the server itself)
 
 about:
-  home: http://www.postgresql.org/
+  home: https://postgresql.org
   license: PostgreSQL
   license_file: COPYRIGHT
```

Comment:
We might want to add a `license_family` to the `meta.yaml` file
https://github.com/AnacondaRecipes/postgresql-feedstock/blob/rebuild_compilers/recipe/meta.yaml#L122&&L123

Reply:
Thanks. I've added `license_family`